src/components/Details/index.js

```diff
@@ -17,7 +17,7 @@ export default function Details() {
 	return (
 		<StyledDetails>
 			<h1>{meal.title}</h1>
-			<Button buttonMode={'goback'} functionToClick={navigate} parameterToClick={'/'}>
+			<Button closeModal functionToClick={navigate} parameterToClick={'/'}>
```

Comment:
If you plan to refactor the codebase in a later US , I would recommend to rename `functionToClick` and `parameterToClick` -  it's a bit confusing.  

By the way I'm not sure if you need to pass these probs at all - in the `<Button>` Component you import the `useNavigate()` hook as well to redirect backt to your home.  🤔

Reply:
Hi Jessica, 
I refactored the Button component with Stefan and he suggested this. 
The idea was to make the component as much reusable as possible, hence the general naming of the props, so that you can give any function you need to the component.
Maybe we could talk about this because I don't really know how I could change it.

Reply:
**Hi @lkk95,**
Ah, I see, if that's what Stefan suggested, then it's fine – too many cooks spoil the broth 😁